.travis.yml

```diff
@@ -1,9 +1,9 @@
-sudo: required
+dist: xenial
```

Comment:
Why is this needed?

Reply:
because of python 3.7:
https://docs.travis-ci.com/user/languages/python/#python-37-and-higher